DebuggerGame/Assets/Scripts/Board Scripts/Board.cs

```diff
@@ -114,16 +114,15 @@ public int GetNumBugs()
     public List<IActionRule> actionRules = new List<IActionRule>();
 
     public List<CollidableObject> collidables;
-    public List<Vector2Int> collidableCoordinates;
+    public Dictionary<Vector2Int, bool> collidableCoordinates;
```

Comment:
It is not very obvious as to what the bool means, but I think it might be easier to just make the `collidables` a map `Dictionary<Vector2Int, CollidableObject>`, and modify `CanEnterCoordinate` accordingly (`bool canPass  = boardObject is Arthropod && collidables[coordinate].bugsCanPass || boardObject is Player && false;`), and modify `Start` too (`foreach(CollidableObject collidable in GetBoardObjectsOfType<CollidableObject>()) {...`)

Also, you should probably initialize the variable _here_ for consistency, instead of `Start` (`public Dictionary<...> collidables = new Dictionary<...>()`)